doc/python_reference.rst

```diff
@@ -946,6 +946,7 @@ Functions that operate on mne-python objects:
    tfr_morlet
    tfr_multitaper
    tfr_stockwell
+   tfr_array
```

Comment:
sorry last minute but is this `tfr_array_morlet` or `tfr_array_multitaper`? I think better to be consistent and explicit ...